src/components/LeftSidebar/Sponsors.astro

```diff
@@ -26,6 +26,13 @@ import UIString from '../UIString.astro';
 			</g>
 		</svg>
 	</a>
+	<a href="https://sentry.io/" aria-label="Sentry">
+		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 44" width="110" height="24">
```

Comment:
LGTM! Just needs an `aria-hidden` on the SVG itself:

```suggestion
		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 44" width="110" height="24" aria-hidden="true">
```

Reply:
Great catch, I even saw that when copying it in and forgot about it when sitting back down to setup the PR 🤦